app/concierge/concierge/migrations/0003_auto_20200105_1221.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.2 on 2020-01-05 12:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('concierge', '0002_auto_20200105_1131'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='apartment',
+            name='tenant',
+            field=models.ForeignKey(null=True, on_delete=models.SET('free'), to='concierge.Tenant'),
+        ),
+    ]
```

Comment:
migration files you do not need to commit. a person who has pulled your code will create them by himself

Reply:
Thanks. I didn't know that and no one explained! I added migrations in all ignore files.